client/state/selectors/get-email-forwards.js

```diff
@@ -11,6 +11,28 @@ export function getEmailForwards( state, domainName ) {
 	return state?.emailForwarding?.[ domainName ]?.forwards ?? null;
 }
 
+/**
+ * Determines whether we are in the process of adding a new email forward to the specified domain.
+ *
+ * @param  {object} state    Global state tree
+ * @param  {string} domainName domainName to request email adding forwards state
+ * @returns {boolean}          adding forwards
+ */
+export function isAddingEmailForward( state, domainName ) {
+	return state?.emailForwarding?.[ domainName ]?.addingForward ?? null;
```

Comment:
Should we not default to `?? false` given that we're declaring the return value to be a `boolean`?
```suggestion
	return state?.emailForwarding?.[ domainName ]?.addingForward ?? false;
```